src/ws/session-utils.c

```diff
@@ -659,7 +659,10 @@ btmp_log (const char *username,
     .ut_type = LOGIN_PROCESS,
   };
 
+  /* see utmp(5), it is ok to not null-terminate these if they have maximum size */
+  /* coverity[buffer_size_warning : FALSE] */
```

Comment:
This is an obnoxious bug in Coverity.  I'd expect to see this for lastlog, but not for utmp.  Or maybe the glibc on the testing machine is just old?

See also https://sourceware.org/bugzilla/show_bug.cgi?id=24899 and https://sourceware.org/bugzilla/show_bug.cgi?id=25844

Reply:
They are running on RHEL 8, with glibc 2.28. The fix only landed in 2.30. Thanks for the reference, I added it as a comment.